builtins.go

```diff
@@ -951,6 +952,73 @@ func builtinStrReplace(i *interpreter, trace traceElement, strv, fromv, tov valu
 	return makeValueString(strings.Replace(sStr, sFrom, sTo, -1)), nil
 }
 
+func base64DecodeGoBytes(i *interpreter, trace traceElement, str string) ([]byte, error) {
+	var sanityCheck = func(v int) (string, bool) {
+		if !(v < 256 && v >= 0) {
+			msg := fmt.Sprintf("encountered invalid codepoint value (must be 0 <= X <= 255), got %d", v)
+			return msg, false
+		}
+
+		return "", true
+	}
+
+	strLen := len(str)
+	if strLen%4 != 0 {
+		msg := fmt.Sprintf("input string appears not to be a base64 encoded string. Wrong length found (%d)", strLen)
+		return nil, makeRuntimeError(msg, i.getCurrentStackTrace(trace))
+	}
+
+	for _, r := range []rune(str) {
+		n := int(r)
+		msg, ok := sanityCheck(n)
```

Comment:
Not sure if this check is required on this side. I think anything outside of the base64 alphabet (which contains only 64 characters from ASCII) is going to cause an error with 
`base64.StdEncoding.DecodeString(str)` - a much stricter condition than this (https://play.golang.org/p/H3_00J8w7RM).

Reply:
ok